src/test/store/receive-profile.test.js

```diff
@@ -1375,22 +1375,20 @@ describe('actions/receive-profile', function () {
       expect(reportError.mock.calls).toMatchSnapshot();
     });
 
-    it('fails if a completely unknown file is passed in', async function () {
+    it('fallback behavior if a completely unknown file is passed in', async function () {
       const invalidJSON = 'invalid';
-      const { args, reportError } = await configureFetch({
+      const { args } = await configureFetch({
         url: 'https://example.com/profile.unknown',
         content: encode(invalidJSON),
       });
 
-      let userFacingError;
+      let userFacingError = null;
       try {
         await _fetchProfile(args);
```

Comment:
Maybe you could put the content in a variable and check the return value of `_fetchProfile`, what do you think?

Reply:
Now asserting the `responseType`.